src/withAutofillProps.spec.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import withAutofillProps from './withAutofillProps'
+import { equal } from 'assert'
+
+describe('withAutofillProps', () => {
```

Comment:
Ok, so about how to test this more, this is what I would do:

```javascript
  it('has the autofilled prop when autofill animation gets triggered', done => {
    const root = document.createElement('div')
    let animationStartCallback
    function Target ({autoFilled, onAnimationStart}) {
      animationStartCallback = onAnimationStart

      return <div>{autoFilled ? 'autofilled' : 'not autofilled'}</div>
    }

    const DecoratedTarget = withAutoFillProps({
      autofilled: true,
    })(Target)

    render(<DecoratedTarget />, root)

    equal(root.innerText, 'not autofilled')

    animationStartCallback('onAutofillStart')

    equal(root.innerText, 'autofilled')
  })
```

…and the equivalent for when the autofilled gets removed ( `animationStartCallback('onAutofillStop')` )

This way you don't have to trigger the _real_ autofill to test it, which is of course impossible 😅